### PR TITLE
Add styles and functionality for image galleries

### PR DESCRIPTION
**Pull request by josephco**
Tied to https://github.com/CasperSleep/nightshade/pull/67, which should be merged after this

### CODE REVIEW

File: src/gallery/Gallery.js [hunk 1]
```diff
@@ -12,18 +13,39 @@ export const Gallery = {
...
       new Flickity(el, options);
     });
   },
+
+  /**
+   * Sets custom imgix settings for gallery images, instantiating Flickity and
+   * fading in the gallery once the first image has loaded
+   * @returns {void}
+  */
+  setupImageGalleries() {
```
Comment (misscs) [inline src/gallery/Gallery.js:38]: This is great! addresses this card that's been in the icebox :D https://trello.com/c/MnLS6IIJ

File: src/gallery/_flickity.scss [hunk 2]
```diff
@@ -1,141 +1,113 @@
-/*! Flickity v1.2.1
-http://flickity.metafizzy.co
----------------------------------------------- */
+/**
+ * @file Flickity
+ * @module Gallery
+ * @overview Styles for custom Flickity controls (based on v1.2.1 http://flickity.metafizzy.co)
```
Comment (misscs) [inline src/gallery/_flickity.scss:4]: :+1: Nice attention to detail to include the version number

File: src/gallery/gallery.macros.html [hunk 3]
```diff
@@ -1,12 +1,25 @@
...
+
+{#
+  * Renders a Gallery image for use in a carousel
+  * @param {string} Gallery image imgix path
+  * @param {string} Image description alt text (optional)
+  * @param {string} Class name for imgix fluid targeting (optional, default: 'imgix-fluid')
+  * @param {string} Class name for style or JS targeting (optional)
+  * @param {string} ID for JS targeting (optional)
+#}
+{% macro gallery_image(image_path, image_description=null, imgix_class='imgix-fluid--gallery', class=null, id=null) %}
```
Comment (misscs) [inline src/gallery/gallery.macros.html:23]: I'd like to get input from the team on what to call these macros. My brain instinctively wants to call this a gallery slide. Maybe it is clear enough to others.

**Comment by misscs** [comment]
No fixes from me. DTM.

File: src/gallery/Gallery.js (hunk 1 above)
Comment (josephco) [inline src/gallery/Gallery.js:38]: Ah nice! This only does it in the instance of the gallery, but if we want to make it something we can apply more globally we could.

**Pull request merged**